src/cdk-experimental/scrolling/auto-size-virtual-scroll.ts

```diff
@@ -106,11 +104,20 @@ export class AutoSizeVirtualScrollStrategy implements VirtualScrollStrategy {
   /** Called when the length of the data changes. */
   onDataLengthChanged() {
     if (this._viewport) {
-      this._updateTotalContentSize();
       this._renderContentForOffset(this._viewport.measureScrollOffset());
     }
   }
 
+  /** Called when the range of items rendered in the DOM has changed. */
+  onContentRendered() {
```

Comment:
Name this function for what it does instead of when it's called?

Reply:
I see these more as hooks for someone implementing a `VirtualScrollStrategy`, therefore I think it makes sense to name them based on when they're called. What it actually does will depend on the particular concrete strategy

Reply:
Would it make sense to then break the content of this function out into another function that gets called here? With the current name/behavior, it's not immediately clear what the code is meant to do without jumping around the file.